### PR TITLE
Add PodDashboardInventoryCard component

### PR DESCRIPTION
**Pull request by mareklibra**
Depends on:
- [x] #2796
- [x] #2601
- [x] #2832

Just the last commit is relevant.

---

podInventory

### CODE REVIEW

**Comment by mareklibra** [comment]
Rebased

File: frontend/public/components/pod-dashboard-inventory.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  // initContainers are not included
+  const containerCount = getPodContainers(pod).length;
+  const volumeCount = getPodVolumes(pod).length;
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Inventory</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody isLoading={isLoading}>
```
Comment (rawagner) [inline frontend/public/components/pod-dashboard-inventory.tsx]: this loading is not needed as you have loadings for specific inventory items

Comment (mareklibra) [reply frontend/public/components/pod-dashboard-inventory.tsx]: removed

**Comment by mareklibra** [comment]
@spadgett , @andybraren , IIUC the Pod Dashboard will probably be descoped for 4.3 and redesigned.

Anyway, this PR is ready based on former design. Would it make sense to land it and make the changes once the new design will exist?

**Comment by mareklibra** [comment]
The pod-dashboard has been descoped for 4.2.

**Pull request closed without merging**